gen_import_utils.py

```diff
@@ -27,6 +28,36 @@ def unique_ordered_list(input_list):
     return unique_elements
 
 
+def set_slashes(path_string):
```

Comment:
these functions are from reading in paths from the json and converting the slashes to either forward or backward for windows or ubuntu systems. os.path.sep